src/components/popover/popover.md

```diff
@@ -0,0 +1,42 @@
+The `popover` component provides functionality somewhere between a modal and a toggletip. It doesn't interrupt flow like a modal would, but provides more than the purely informational function of a toggletip. It is a way to allow for greater branching functionality while staying in-flow.
+
+The `popover` component provides a ref for getting and setting the `open` state, as well as an `onClose` callback in props. These features can be used to more tightly control the open/close functionality, and to return focus to the correct element once closed.
+
+`popover`s will close if a `click` event reaches the root `window`, so click events that should not close the `popover` should have `stopPropogation` called on them.
+
+```js
+const popoverRef = React.createRef();
+
+const onButtonClicked = () => {
+  popoverRef.current.open = !popoverRef.current.open;
+}
+
+<React.Fragment>
+  <button onClick={onButtonClicked}>Toggle Popover</button>
```

Comment:
I couldn't get the first example to pop-up on staging.

Reply:
@bennapp It was probably just lower on the screen than you expected, will you look again?

From the DRD:
> If user scrolls page behind popover, the popover should remain fixed in place and not move with the page.

Reply:
Still can't find it 🤷  👻  I'm looking here: https://mavenlink.github.io/design-system/webinf-all-about-the-popover-175407314/#/Components/Popover

at the first 'Toggle Popover'

Reply:
Strange, I'm seeing it! What browser are you using to test? Maybe a hard reload?